src/project/project.controller.ts

```diff
@@ -220,14 +219,46 @@ export class ProjectController {
     );
   }
 
+  @UseGuards(...IsEmailVerifiedGuard)
+  @ApiCookieAuth()
+  @Put(':id/invitation/accept')
+  async acceptEnrollRequest(
```

Comment:
We might need to modify this method name to 'acceptEnrollInvitation' in order to match what is being called from the service (better readability)